### ISSUE TITLE
How to get the data points per peak?

### ISSUE THREAD

**Issue by maganam5** [vdemichev/DiaNN#745]
Hi Vadim! 
I hope you are well! 

Is there any way of getting the data points per peaks for the MS2 scans? Ideally, we'd like to understand and look at this for peptide features. Would there be any of reporting on this?

I saw if our scan window was set to 0 this would be "Peak width"... say our 
scan window to is 6. I does this mean this no longer correspond to the estimate of our data points per peak? Also we have one "Peak width: 3.14" in our log files. Is this average across all files? 

Thank you so much in advance!

**Comment by vdemichev** [comment]
Yes, in beta 22 (included in QuantUMS preprint) the .parquet report files contains 'FWHM' column, which is the peak width at half maximum, you can then calculate dp/peak since you know the DIA cycle time.

Yes, what is printed as "Peak width: 3.14" in the log is representative of the average dp/peak at FWHM.

**Comment by vdemichev** [comment]
Not sure yet, there are a lot of changes & this needs thorough testing. Maybe a pre-release in a month or two, and then labelled as a release if no issues are detected by the community.
